services/web/server/src/simcore_service_webserver/products/_api.py

```diff
@@ -48,7 +55,7 @@ def _themed(dirname: str, template: str) -> Path:
 
 
 async def _get_content(request: web.Request, template_name: str):
-    repo = ProductRepository(request)
+    repo = ProductRepository.create_from_request(request)
```

Comment:
this one does not have the above note, is the change intended? Should it have been `create_from_app`?

Reply:
this one is created from a request. it's fine.